tests/v2/validation/publicapi/clusterroletemplatebindings/crtb_create_steve_test.go

```diff
@@ -0,0 +1,307 @@
+
+	// Validate errors when user input existing crtbName while creating crtb
+	_, err = client.Steve.SteveType(crtbApiEndPoint).Create(inputData)
+	require.Error(t, err)
+	require.ErrorContains(t, err, "409 Conflict")
+	require.ErrorContains(t, err, "AlreadyExists")
+}
+
+func validateErrorsWhileCreatingCrtbBySteveWhenUserInputNotExistedRoleTemplateName(t *testing.T, client *rancher.Client, remoteClusterID string, readJson []byte) {
+	newUser, err := users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(t, err)
+	crtbName := namegen.RandStringLower(8)
+
+	inputData := unmarshalReadJson(t, readJson, remoteClusterID, crtbName, newUser.ID, defaultRoleTemplateName)
+	require.NoError(t, err)
+	inputData["roleTemplateName"] = "roleTemplateNameNotExisted"
+
+	_, err = client.Steve.SteveType(crtbApiEndPoint).Create(inputData)
+	require.Error(t, err)
+	require.ErrorContains(t, err, "400 Bad Request")
+}
+
+func validateErrorsWhileCreatingCrtbBySteveWhenUserInputClusterNameAndNameSpaceNameDifferentAndNotExisted(t *testing.T, client *rancher.Client, remoteClusterID string, readJson []byte) {
+	newUser, err := users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(t, err)
+	crtbName := namegen.RandStringLower(8)
+
+	inputData := unmarshalReadJson(t, readJson, remoteClusterID, crtbName, newUser.ID, defaultRoleTemplateName)
+	require.NoError(t, err)
+	inputData["clusterName"] = "local"
+
+	_, err = client.Steve.SteveType(crtbApiEndPoint).Create(inputData)
+	require.Error(t, err)
+	require.ErrorContains(t, err, "400 Bad Request")
+}
+
+func validateErrorsWhileCreatingCrtbBySteveWhenUserInputNotExistedNameSpaceName(t *testing.T, client *rancher.Client, remoteClusterID string, readJson []byte) {
+	newUser, err := users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(t, err)
+	crtbName := namegen.RandStringLower(8)
+
+	inputData := unmarshalReadJson(t, readJson, remoteClusterID, crtbName, newUser.ID, defaultRoleTemplateName)
+	require.NoError(t, err)
+	inputData["metadata"].(map[string]interface{})["namespace"] = "namespacenotexisted"
+	inputData["clusterName"] = "namespacenotexisted"
+
+	_, err = client.Steve.SteveType(crtbApiEndPoint).Create(inputData)
+	require.Error(t, err)
+	require.ErrorContains(t, err, "404 Not Found")
+}
+
+func validateErrorsWhileCreatingCrtbBySteveWhenUserInputClusterNameAndNameSpaceNameDifferentAndExisted(t *testing.T, client *rancher.Client, remoteClusterID string, readJson []byte) {
+	newUser, err := users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(t, err)
+	crtbName := namegen.RandStringLower(8)
+
+	inputData := unmarshalReadJson(t, readJson, remoteClusterID, crtbName, newUser.ID, defaultRoleTemplateName)
+	require.NoError(t, err)
+	inputData["clusterName"] = "local"
+
+	_, err = client.Steve.SteveType(crtbApiEndPoint).Create(inputData)
+	require.Error(t, err)
+	require.ErrorContains(t, err, "Forbidden: clusterName and namespace must be the same value")
```

Comment:
nit; will this error message stay relatively the same? It seems long, and may break in the future because of that.